composer.json

```diff
@@ -4,7 +4,7 @@
     "type": "library",
     "license": "GPL-2.0-only",
     "require": {
-        "php": ">=7.2.0",
+        "php": ">=7.4.0",
```

Comment:
This library is a public one, and while PHP 7.2 is obviously no-longer supported on its own, doing this is a breaking change.

I'd preferred to have a dedicated PHP version support upgrade channel to the lti1p3 libraries.

Reply:
i actually consider it as a breaking change and planed to bump major version of library. But if it require some more explicit procedure. i will rollback code to compatible with 7.2 version